blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md

```diff
@@ -0,0 +1,261 @@
+pod/cert-manager-webhook-bc964d98b-mkjj7         1/1     Running       0          45m
+
+NAME                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+service/apisix-admin                ClusterIP   10.96.16.25     <none>        9180/TCP                     57m
+service/apisix-etcd                 ClusterIP   10.96.232.251   <none>        2379/TCP,2380/TCP            57m
+service/apisix-etcd-headless        ClusterIP   None            <none>        2379/TCP,2380/TCP            57m
+service/apisix-gateway              NodePort    10.96.118.75    <none>        80:32039/TCP,443:30107/TCP   57m
+service/apisix-ingress-controller   ClusterIP   10.96.13.76     <none>        80/TCP                       57m
+service/cert-manager-webhook        ClusterIP   10.96.182.188   <none>        443/TCP                      45m
+
+NAME                                        READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/apisix                      1/1     1            1           57m
+deployment.apps/apisix-ingress-controller   1/1     1            1           57m
+deployment.apps/cert-manager                1/1     1            1           45m
+deployment.apps/cert-manager-cainjector     1/1     1            1           45m
+deployment.apps/cert-manager-webhook        1/1     1            1           45m
+
+NAME                                                   DESIRED   CURRENT   READY   AGE
+replicaset.apps/apisix-5d99956d88                      1         1         1       63s
+replicaset.apps/apisix-69459554d4                      0         0         0       57m
+replicaset.apps/apisix-ingress-controller-74c6b5fbdd   0         0         0       57m
+replicaset.apps/apisix-ingress-controller-7b5c767cc7   1         1         1       55m
+replicaset.apps/apisix-ingress-controller-7d58db957c   0         0         0       55m
+replicaset.apps/cert-manager-5ffd4f6c89                1         1         1       45m
+replicaset.apps/cert-manager-cainjector-748dc889c5     1         1         1       45m
+replicaset.apps/cert-manager-webhook-bc964d98b         1         1         1       45m
+
+NAME                           READY   AGE
+statefulset.apps/apisix-etcd   2/3     57m
+
+NAME                                     COMPLETIONS   DURATION   AGE
+job.batch/cert-manager-startupapicheck   1/1           6m24s      45m
+```
+
+>[Kubernetes Controller Manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) 的机制决定了 Pod 名称会有所不同。
+
+## 步骤四：申请证书并测试
+
+首先我们需要配置证书颁发对象。
+
+```yaml
+# issuer.yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: issuer
+  namespace: ingress-apisix
+spec:
+  selfSigned: {}
+```
+
+并创建自签名证书颁发者。
+
+```bash
+kubectl apply -f issuer.yaml
+```
+
+>请注意，自签名颁发对象不推荐使用在生产环境中！更多证书颁发对象的配置请参考[这里](https://cert-manager.io/docs/configuration/)。
+然后为域名 `httpbin.org` 创建一张证书。
```

Comment:
```suggestion

然后为域名 `httpbin.org` 创建一张证书。
```

Reply:
See, if you don't add a new line, it understands the two sentences as whole, which is not the case.

<img width="991" alt="image" src="https://user-images.githubusercontent.com/36651058/217521148-22a38126-34fb-4768-8306-46f62583b3a1.png">

Reply:
yes , it is,  thanks